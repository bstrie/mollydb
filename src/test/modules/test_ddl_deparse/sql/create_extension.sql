---
--- CREATE_EXTENSION
---

CREATE EXTENSION mdb_stat_statements;
