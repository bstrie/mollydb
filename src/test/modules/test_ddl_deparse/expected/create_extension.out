---
--- CREATE_EXTENSION
---
CREATE EXTENSION mdb_stat_statements;
NOTICE:  DDL test: type simple, tag CREATE EXTENSION
